### PR TITLE
Project Portfolio Updated

### PR DESCRIPTION
**Pull request by for-ashraf**
In this Pull request I tried to incorporate all the required changes in the project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,27 +8,166 @@
...
         </div>
         <button type="button" class="colbtn">Start Collaboration</button>
-        <div class="dcmg">
+        <div class="decoimg">
+        </div>
+    </section>
+
+    <section class="works">
+        <div class="headtitle">
+            <h1 class="mttitle">Projects</h1>
```
Comment (jaflih) [inline index.html]: [OPTIONAL] It is recommended to have only one h1 for each page, because you use one before, use h2 here

File: CSS/styles.css [hunk 2]
```diff
@@ -38,7 +36,11 @@ body {
   align-items: center;
 }
 
-.htitle {
+.headtitle {
+  width: 100%;
+}
```
Comment (jaflih) [inline CSS/styles.css]: Please use flexbox here to layout the element.
This is part of the requirement
image

File: CSS/styles.css [hunk 3]
```diff
@@ -96,3 +104,179 @@ body {
...
+.projtxt {
+  margin-bottom: 0.5em;
+  padding-left: 1em;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 20px;
+  line-height: 80%;
+  color: #fff;
+}
```
Comment (jaflih) [inline CSS/styles.css:137]: Please correct the design to stick to Figma

here the problem is that the h2 has a margin-top so the element make a white space.
image

Figma:
image

File: index.html [hunk 4]
```diff
@@ -8,27 +8,166 @@
...
+                    </ul>
+                </div>
+                <button type="button" class="projbtn">Check out this Projects<img src="./Images/arrow.png" alt=""></button>
+            </div>
+        </div>
+    </section>
+
+    <section class="about">
+        <div class="headtitle">
+            <h1 class="mttitle" >About</h1>
```
Comment (jaflih) [inline index.html]: [OPTIONAL]Please use h2 instead of h1 like the previous comment

File: index.html [hunk 5]
```diff
@@ -8,27 +8,166 @@
...
+            </div>
+            <div class="litxt">
+            <ul>
+                <li>Codekit</li>
+                <li>Codepen</li>
+                <li>GitHub</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+            </ul>
+            </div>
```
Comment (jaflih) [inline index.html]: Good job for using ul/li for listing the specificity like Codekit, Codepen ...

For the same reason, you have to use ul/li for listing these 3 elements :  Languages ,Frameworks ,Skills

**Review by jaflih — CHANGES_REQUESTED** [review]
Hi @for-ashraf, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✔️ Linters passed 💯
✔️ The content is aligned with flexbox and grid🥇 
✔️ Great commit history

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @jaflih in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 5 above)
Comment (for-ashraf) [reply index.html]: Hello, Languages, Frameworks, and Skills are the titles themselves and each in separate div. Making it li will effect the design, pleaes

File: index.html [hunk 6]
```diff
@@ -8,27 +8,166 @@
...
```
Comment (tresorsawasawa) [inline index.html]: - [ ] As required by the previous reviewer, Kindly make sure you use **ul/li** for listing these 3 elements: **Languages** ,**Frameworks**, and **Skills**. As you did by listing the specificity like Codekit, Codepen,...

**Review by tresorsawasawa — COMMENTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @for-ashraf 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required changes  :recycle::recycle: 
- [ ] **Check the comments under the review**.

## Optional suggestions
- [ **OPTIONAL** ] Kindly make sure to improve the PR description by listing those project's requirements.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — APPROVED** [review]
Hi @for-ashraf, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**